packages/landingpage/utils/routes.tsx

```diff
@@ -48,8 +48,7 @@ export enum SubRoutes {
 }
 
 const productionModeExcludedSubroutes: { [key: string]: string[] } = {
-  library: ['patterns'],
-  // 'anotherMainRoute': ['subroute1', 'subroute2']
```

Comment:
I think we need some more patterns (and also functionality in #1165) to release it to the public, so we might want to keep it disabled for now